Source/Core/Core/TitleDatabase.h

```diff
@@ -8,6 +8,8 @@
 #include <unordered_map>
 
 #include "Common/CommonTypes.h"
+#include "Common/Lazy.h"
+#include "DiscIO/Enums.h"
```

Comment:
This can likely be replaced with a forward declaration (forward declared enums are still complete types, so it will work, despite the enum type being used as a key in one of the unordered maps).